app/views/api_new/supporters/_supporter.json.jbuilder

```diff
@@ -3,7 +3,7 @@
 # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
 # Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
 
-json.(supporter, :name, :organization, :phone, :anonymous, :deleted)
+json.(supporter, :name, :email, :organization, :phone, :anonymous, :deleted)
```

Comment:
Can you make this commit a separate PR as well as add tests for any place this is currently used?